dfdaemon/downloader/downloader.go

```diff
@@ -16,11 +16,16 @@
 
 package downloader
 
+import "context"
+
 // Interface specifies on how an plugin can download a file.
 type Interface interface {
 	// Download download url file to file name
 	// return dst path and download error
 	Download(url string, header map[string][]string, name string) (string, error)
```

Comment:
I am wondering if we could directly remove `Download(url string, header map[string][]string, name string) (string, error)` . @Starnop @yeya24 @lowzj

Reply:
Agree.

Reply:
Removed. I'm hesitate to rename back to `Download` as it is not future-proof yet(I mean, I'm not certain that every implementation can be implemented with Context).

Reply:
I think it's better to rename back to `Download`. 😄

Reply:
@zhouhaibing089 WDYT? And can we make it move on?

Reply:
I'd like to keep it as it is just like `exec.Command` and `exec.CommandContext` unless you are strong on it.